src/api/BioLink.js

```diff
@@ -393,6 +443,8 @@ export function getNodeAssociations(nodeType, identifier, cardType, params) {
           reject(responseData);
         }
         else {
+          // console.log('getNodeAssociations', nodeType, identifier, cardType, url);
```

Comment:
clean out

Reply:
Until BL stabilizes, I'm often debugging its response and these console.log are worth having around, so I try to keep them commented out, but in the code.